src/api/app/views/webui/request/beta_show_tabs/_mentioned_issues.html.haml

```diff
@@ -1,5 +1,15 @@
-%p
-  The mentioned issues aren't here yet, but they will be displayed here in the future.
+- if issues.empty?
+  %p No issues are mentioned for this request action.
```

Comment:
Since the `Mentioned Issues` tab has a badge displaying the number of mentioned issues for the request action, I prefer to display the tab. I think this follows well the [principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment). This way, users know at a quick glance if the request action has mentioned issues or not. If the tab is not there, they have to know this unwritten rule. They might even be looking on the page for the mentioned issues without knowing that there are simply none for the request action.